process/templates/WG_NAME_graduation_stage.md

```diff
@@ -1,11 +1,36 @@
 ## Working Group graduation application
 
+### WG has met all Incubating requirements
+  * "link to Incubating PR"
+
 ### List of regular contributors
 The WG must have at least 5 contributors from at least 3 different organizations attending regularly as recorded in meeting minutes.
-
   * "name, affiliation, GitHub ID"
 
-### Working Group activity
-The WG must have met at least 4 times over a period of at least 2 months since becoming `Incubating`
+### Release cadence
+The TI must be able to show a consistent release cadence.
+  * "link to release documentation"
+
+### Governance
```

Comment:
Looks like these were copied over from the Projects doc?

Reply:
it was, but the idea is the same:  we want our WGs to continue to collaborate and work together and periodically deliver new or updated things to the community